modules/Makefile.kubernetes

```diff
@@ -68,6 +74,15 @@ kubernetes\:info:
 	@echo -e "SSH Tunnel: $(call yellow,$(KUBECTL_SSH_TUNNEL))"
 	@echo -e "SSH User: $(call yellow,$(KUBECTL_SSH_USER))"
 
+# (private) Validate a configuration
+kubernetes\:validate:
+	@for env in $$(grep -Eo '\$$[A-Z_0-9]+' $(KUBERNETES_RESOURCE_PATH)/$(KUBERNETES_APP)-configmap.yml | cut -d\$$ -f2); do \
+    if [ -z "$${!env}" ]; then \
+      echo "$$env not set"; \
```

Comment:
nit: `$$env not defined in $(KUBERNETES_RESOURCE_PATH)/$(KUBERNETES_APP)-configmap.yml`